catalog/ESA/ESA_WorldCereal_2021_MODELS_v100.jsonnet

```diff
@@ -0,0 +1,133 @@
+    
+    The collection should be filtered using one or more of the following image properties:
+    - **aez_id**, holding the ID of the AEZ to which the image belongs
+    - **product**, describing the WorldCereal product name of the image
+    - **season**, describing for which season the image is valid.
+    
+    **References:**
+    [1] [WorldCereal methodology and products paper](https://doi.org/10.5194/essd-2023-184)
+    [2] Need to reference the AEZ FeatureCollection here.
+    [3] [WorldCereal global seasonality paper](https://doi.org/10.1080/15481603.2022.2079273)
+  |||,
+  license: license.id,
+
+  links: ee.standardLinks(subdir, id) + [
+    ee.link.license(license.reference)],
+  keywords: ['agriculture', 'copernicus', 'crop', 'esa', 'global', 'landcover', 'landsat', 'sentinel1_derived', 'sentinel2_derived'],
+  providers: [
+    ee.producer_provider(
+      'ESA WorldCereal Consortium',
+      'https://esa-worldcereal.org/en'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent_global('2020-01-01T00:00:00Z', '2021-12-31T23:59:59Z'),
+  summaries: {
+    'gee:schema': [
+      {
+        name: 'aez_id',
+        description: 'ID of the AEZ to which the product belongs.',
+        type: ee_const.var_type.int
+      },
+      {
+        name: 'product',
+        description: 'WorldCereal product name.',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'season',
+        description: 'Season for which the product is valid.',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'start_date',
+        description: 'Start date (yyyy-mm-dd) for which the product is valid.',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'end_date',
+        description: 'End date (yyyy-mm-dd) for which the product is valid.',
+        type: ee_const.var_type.string
+      },
+    ],
+    gsd: [10],
+    'eo:bands': [
+      {
+        name: 'classification',
+        description: 'Classification'
+      },
+      {
+        name: 'confidence',
+        description: 'Confidence'
+      },
+    ],
+    classification: {minimum: 0.0, maximum: 100.0, 'gee:estimated_range': false},
+    confidence: {minimum: 0.0, maximum: 100.0, 'gee:estimated_range': false},
+    'gee:visualizations': [{
+      display_name: 'Classification confidence',
+      lookat: {lon: 5.02666, lat: 50.63096, zoom: 10},
+      image_visualization: {
+        band_vis: {
+          min: [0],
+          max: [100],
+          palette: [
+            'be0000','fff816','069711'
+          ],
+          bands: ['confidence'],
+        },
+      },
+    }],
+  },
+  'sci:doi': '10.5194/essd-2023-184',
+  'sci:citation': |||
+    Van Tricht, K., Degerickx, J., Gilliams, S., Zanaga, D., Battude, M., Grosu, A., ... & Szantoi, Z. (2023). WorldCereal: a dynamic open-source system for global-scale, seasonal, and reproducible crop and irrigation mapping. Earth System Science Data Discussions, 2023, 1-36.
```

Comment:
the page https://doi.org/10.5194/essd-2023-184 suggests the citation without  "..." - why not use the full text here?

Van Tricht, K., Degerickx, J., Gilliams, S., Zanaga, D., Battude, M., Grosu, A., Brombacher, J., Lesiv, M., Bayas, J. C. L., Karanam, S., Fritz, S., Becker-Reshef, I., Franch, B., Mollà-Bononad, B., Boogaard, H., Pratihast, A. K., and Szantoi, Z.: WorldCereal: a dynamic open-source system for global-scale, seasonal, and reproducible crop and irrigation mapping, Earth Syst. Sci. Data Discuss. [preprint], https://doi.org/10.5194/essd-2023-184, in review, 2023.